examples/SyncMove/SyncMove.ino

```diff
@@ -0,0 +1,46 @@
+// Basic example of making a STS3215 servos move async.
```

Comment:
rather *two servos* move *synchronously*   - async is already all we had before.

Reply:
Thanks for your review and sorry for the formatting. I already fixed it. Moreover, to make it easier I added several functionalities such as setTargetAcceleration, setId, setPositionOffset, setTargetPositions, and added speed to setTargetPosition. currently, that's all of the changes.

Reply:
Great, I'll merge this and make a new release.

Reply:
Great thanks.